recipe/run_test.py

```diff
@@ -1,7 +1,8 @@
 import curses
 import sys
 
-if __name__ == '__main__':
+# The Python test on OS X will not pass until Python has been recompiled against ncurses 6.0
+if not sys.platform == 'darwin':
```

Comment:
I don't think this is a matter of osx. It affects linux too, but I guess there's a system ncurses 5.9 lying around which is used in the test phase.

Reply:
Yep. I left it running on Linux b/c of that. But we should revert that once we get Python re-built.